### ISSUE TITLE
Clicking on LDAP users in backend throws error

### ISSUE THREAD

**Issue by dnaman182** [snipe/snipe-it#1182]
After importing users with the LDAP tool, going in to the backend and then clickin on a username throws an error. 

Error:
ErrorException (E_ERROR)
HELP
Trying to get property of non-object (View: C:\Apache24\htdocs\snipe-it\app\views\backend\users\view.blade.php)
![image](https://cloud.githubusercontent.com/assets/5384313/9966242/012bc764-5df9-11e5-8ad6-dc13dc6d56fa.png)

URL:
127.0.0.1/admin/users

Please let me know if you need any more data. I'm going to start looking at the issue myself and will post a resolution if I find one.

**Comment by dnaman182** [comment]
Issue was caused by a workaround I made for importing LDAP users. Could be somewhat related as LDAP import didn't list any locations so I simply removed it... this error happens when location_id in the database is not equal to NULL.

**Comment by snipe** [comment]
Are you using the version that asks you select a location from the drop down before importing? (Latest on develop)

**Comment by dnaman182** [comment]
Yes. I see now that you can add a location through the backend, but I was unaware of this when I did it so I just hardcoded a "1". Any user with the 1 would throw an error in when being clicked on in the backend. This issue was an issue of me not knowing how to use the software.

Could be beneficial to include a note to add a location through the backend before importing?

**Comment by snipe** [comment]
I'm not sure I understand. If you select the location and then do the LDAP import, the location you select will be applied to the users. There isn't anything to hardcode.

**Comment by snipe** [comment]
(Also, throwing an error with an invalid location was fixed with this commit: https://github.com/snipe/snipe-it/commit/611da2ffbd1f8fd24a11ac9b152efb7a620a4ee7)

**Comment by dnaman182** [comment]
Correct. I didn't know to add a location and didn't see it so I decided to do it myself in the code. This really isn't an issue and was just my mistake, I didn't know that until after which is why I closed the issue.

**Comment by snipe** [comment]
No, I get that - I'm trying to figure out how you didn't see it though. Are you using an older version where the location dropdown on the LDAP import page wasn't present? (I'm trying to determine if this is an issue that has already been fixed by adding the location dropdown, or if it's a UX issue where people aren't noticing the dropdown.)

**Comment by dnaman182** [comment]
I went to the LDAP import page and saw this: 
![image](https://cloud.githubusercontent.com/assets/5384313/9972926/506c5b84-5e28-11e5-9c63-88b606be8cde.png)

I didn't know how to set a location and I tried for a bit (I know now) and then decided "OK, I'll just code a workaround". In this case I didn't know how to add values to the location dropdown. I think if there was a link on the word "location" in "You will need to have at least one location set to use this feature" to the location page I would have figured it out.
